src/screens/Feed/components/PostCreator/PostCreator.js

```diff
@@ -253,8 +253,15 @@ class PostCreator extends React.PureComponent {
 
     if (busy) return;
 
-    // Don't allow posting if content and gif is empty
-    if (isEmpty(content)) {
+    /*
+      Force user to add content unless they have uploads or gif set
+      We use `trimStart` and `trimEnd` instead of `trim` so that we can maintain any whitespaces in the content itself,
+      `trim` reduces whitespace to 1 characer.
+      E.g if we had '__a___b____c__' (where _ = space) then `trim` would make that 'a_b_c'
+          where as `trimStart` and `trimEnd` would become 'a___b____c'
+    */
```

Comment:
Does the JS engine have a native `trim()`? It sounds like that implementation correctly just trims the ends https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/trim